`-DJEMALLOC_NO_DEMANGLE` is necessary to avoid a bunch of "implicit declaration" errors

sds.c:409:34: error: implicit declaration of function 'je_nallocx'; did you mean 'nallocx'? [-Wimplicit-function-declaration]
409 |        alloc_already_optimal = (je_nallocx(newlen, 0) == bufsize);
    |                                 ^~~~~~~~~~
    |                                 nallocx
diff --git a/src/Makefile b/src/Makefile
index 3bc9f11c0..a4b23d986 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -229,8 +229,8 @@ endif
 
 ifeq ($(MALLOC),jemalloc)
 	DEPENDENCY_TARGETS+= jemalloc
-	FINAL_CFLAGS+= -DUSE_JEMALLOC -I../deps/jemalloc/include
-	FINAL_LIBS := ../deps/jemalloc/lib/libjemalloc.a $(FINAL_LIBS)
+	FINAL_CFLAGS+= -DUSE_JEMALLOC -DJEMALLOC_NO_DEMANGLE -I/usr/include/jemalloc
+	FINAL_LIBS := -ljemalloc $(FINAL_LIBS)
 endif
 
 # LIBSSL & LIBCRYPTO
